data-ingestion-service/src/main/java/gov/cdc/dataingestion/config/OpenAPIConfig.java

```diff
@@ -1,30 +1,32 @@
 package gov.cdc.dataingestion.config;
 
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
+import io.swagger.v3.oas.models.Components;
 import io.swagger.v3.oas.models.OpenAPI;
 import io.swagger.v3.oas.models.info.Contact;
 import io.swagger.v3.oas.models.info.Info;
-import io.swagger.v3.oas.models.info.License;
+import io.swagger.v3.oas.models.security.SecurityScheme;
 import io.swagger.v3.oas.models.servers.Server;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
 import java.util.List;
 
-
 @Configuration
 public class OpenAPIConfig {
 
     @Bean
     public OpenAPI myOpenAPI() {
         Server devServer = new Server();
-        devServer.setUrl("https://dataingestion.datateam-cdc-nbs.eqsandbox.com");
-        devServer.setDescription("Server URL in Development environment");
+        devServer.setUrl("http://localhost:8081");
+        devServer.setDescription("Server URL in Local environment");
 
-        Server prodServer = new Server();
-        prodServer.setUrl("https://dataingestion.datateam-cdc-nbs.eqsandbox.com");
-        prodServer.setDescription("Server URL in Production environment");
+        Server dts1Server = new Server();
+        dts1Server.setUrl("https://dataingestion.dts1.nbspreview.com");
+        dts1Server.setDescription("Server URL in DTS environment");
+
+        Server int1Server = new Server();
+        int1Server.setUrl("https://dataingestion.int1.nbspreview.com");
```

Comment:
should we have these as environment variable?

Reply:
Update to get the server host from the end variable.